src/app/components/ChartbeatAnalytics/index.test.tsx

```diff
@@ -167,24 +173,20 @@ describe('Charbeats Analytics Container', () => {
     expect(container.firstChild).toBeNull();
   });
 
-  it('should call sendCanonicalChartbeatBeacon when platform is canonical, and toggle enabled for chartbeat on test', () => {
+  it('should add Chartbeat Helmet script with correct config when platform is canonical, toggle enabled and on test in Lite mode', () => {
     process.env.SIMORGH_APP_ENV = 'test';
-    const mockAmp = jest.fn().mockReturnValue('amp-return-value');
-    // @ts-expect-error requires mocking for testing purposes
-    amp.default = mockAmp;
 
     const expectedConfig = {
       uid: 50924,
       domain: 'test-domain',
       idSync: {
         bbc_hid: 'cookie',
       },
-      path: '/',
-      sections: 'secction1 section2',
+      path: '/.lite',
```

Comment:
Could we use a path which is more representative of a real lite URL 
```suggestion
      path: '/pidgin.lite',
```

or 
```suggestion
      path: '/pidgin/articles/c00000000o.lite',
```